client/src/main/resources/templates/js/main.js

```diff
@@ -601,7 +601,7 @@ function displaySource(src, title) {
     $('.btn-screenshot' + buttonAdditonalClassName).click(function() {
       var testcase = data[parseInt($(this).attr('data-index'))];
       var params = ' ';
-      if (testcase.parameters !== null) {
+      if ((testcase.parameters !== null) && (typeof testcase.parameters !== 'undefined')) {
```

Comment:
We can simplify the condition as if (testcase.parameters) will validate the null check and undefined check